app/controllers/api/v1/groups_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Api::V1::GroupsController < ApplicationController
+  def index
+    @groups = Group.all
+
+    render json: @groups
+  end
+
+  def show
+    @group = Group.find(params[:id])
```

Comment:
`@group = Group.find(params[:id])` is repeating in action `update` and `destroy`. For avoiding this repetition, you can create a `before_action` with a private method and put this code inside. Is a tip that a usually use in my projects 😄 .